src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -449,7 +449,8 @@ else if (groupIdx > lastGroup) {
         else {
             log(line, MSG_ORDERING, name);
         }
-        if (checkSeparatorInGroup(groupIdx, isStatic, line)) {
+        if (checkSeparatorInGroup(groupIdx, isStatic, line)
+            || checkSeparatorInStaticImport(isStatic, line)) {
```

Comment:
Just a question, why did you separate this into 2 methods instead of combining them into 1?
They are both named similarly and both take similar parameters.

Reply:
Done.